client/src/main/java/org/apache/celeborn/client/write/DataPusher.java

```diff
@@ -123,6 +123,9 @@ public void run() {
             while (stillRunning()) {
               try {
                 ArrayList<PushTask> tasks = dataPushQueue.takePushTasks();
+                if (Objects.isNull(tasks)) {
```

Comment:
Oh, I just mistaken to return null instead of return tasks in DataPushQueue.takePushTasks. Since tasks is initialized to ArrayList there, we don't need to check null here, WDYT?

Reply:
Yea, I am just verify this way in internal CI